content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -319,7 +319,7 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).
+For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-managed/).
```

Comment:
thank you for the PR, but i think the old page lets the user navigate to a section of the Cilium docs they want, while the new proposed link sends them to managed k8s, which they might not be using.

a potentially better link might be:
https://docs.cilium.io/en/stable/gettingstarted/#installation

/cc @aanm 
/hold 
/sig cluster-lifecycle

Reply:
@neolit123 thanks for the heads up, given this is kubeadm, the link should be:

https://docs.cilium.io/en/stable/gettingstarted/k8s-install-default/